### PR TITLE
Adds holidays, fixes beer day date

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Fixed the Beer Day date and added a few more holidays.
/:cl:

Beer Day start date was wrong (it's the first Friday in August, not August 5th). 
I also feel that Bee/Tea/Independence days are relevant enough for SS13 to add them as well.

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -208,7 +230,15 @@
 	drone_hat = /obj/item/clothing/mask/facehugger/dead
 
 /datum/holiday/UFO/getStationPrefix() //Is such a thing even possible?
-	return pick("Ayy","Truth","Tsoukalos","Mulder") //Yes it is!
+	return pick("Ayy","Truth","Tsoukalos","Mulder","Scully") //Yes it is!
+
+/datum/holiday/USA
```
Comment (Wilchenx) [inline code/modules/holiday/holidays.dm]: nanotrasen doesn't care about nations, does this really fit?

Comment (81Denton) [reply code/modules/holiday/holidays.dm]: There are liberation station vendors and centcom was burger flag colored for the 4th of July, I think it fits. Nothing I feel strongly about either way though

Comment (MrDoomBringer) [reply code/modules/holiday/holidays.dm]: image
i mean we already have plenty of refrences to america so i think its fine

Comment (optimumtact) [reply code/modules/holiday/holidays.dm]: except the headcoder isn't american now.

The oceaniac reign of terror begins

**Review by SpaceManiac — APPROVED** [review]
lgtm

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (optimumtact) [reply code/modules/holiday/holidays.dm]: yes

**Pull request merged**